ods_ci/tasks/Resources/Provisioning/Hive/AZURE/azure-cluster.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: hive.openshift.io/v1
+  kind: ClusterPool
+  metadata:
+    name: {{ hive_cluster_name|default("rhodsazure") }}-pool
+    namespace: {{ hive_claim_ns }}
+  spec:
+    baseDomain: {{ base_domain }}
+    imageSetRef:
+      name: {{ image_set|default("rhods-openshift") }}
+    platform:
+      azure:
+        baseDomainResourceGroupName: {{ azure_domainresourcegroup }}
+        cloudName: AzurePublicCloud
+        region: {{ azure_region|default("eastus") }}
+        credentialsSecretRef:
+          name: azure-creds
+    installConfigSecretTemplateRef:
+      name: azure-sno-install-config
+    size: 0
+    skipMachinePools: true
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: azure-sno-install-config
+    namespace: {{ hive_claim_ns }}
+  type: Opaque
+  stringData:
+    install-config.yaml: |
+      apiVersion: v1
+      baseDomain: {{ base_domain }}
+      compute:
+      - name: worker
+        platform:
+          azure:
+            type: {{ worker_node_instance_type|default("Standard_D8s_v3") }}
+            osDisk:
+              diskSizeGB: 128
+        replicas: {{ worker_node_replicas|default("3") }}
+      controlPlane:
+        name: master
+        platform:
+          azure:
+            type: {{ master_node_instance_type|default("Standard_D8s_v3") }}
+            osDisk:
+              diskSizeGB: 128
+        replicas: {{ master_node_replicas|default("3") }}
+      metadata:
+        name: azure-sno
+      networking:
+        clusterNetwork:
+        - cidr: 10.128.0.0/14
+          hostPrefix: 23
+        machineNetwork:
+        - cidr: 10.0.0.0/16
+        networkType: {{ ocp_network_type|default("OVNKubernetes") }}
+        serviceNetwork:
+        - 172.30.0.0/16
+      fips: {{ fips_validation|default("false") }}
+      platform:
+        azure:
+          baseDomainResourceGroupName: {{ azure_domainresourcegroup }}
+          cloudName: AzurePublicCloud
+          region: {{ azure_region|default("eastus") }}
+      pullSecret: {{ pull_secret|default("") }}
+      sshKey: {{ ssh_key|default("") }}
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: azure-creds
+    namespace: {{ hive_claim_ns }}
+  type: Opaque
+  stringData:
+    osServicePrincipal.json: |
+      {
+        "subscriptionId": {{ '\"' + azure_subscriptionId + '\"' }},
+        "clientId": {{ '\"' + azure_clientId + '\"' }},
+        "clientSecret": {{ '\"' + azure_clientSecret + '\"' }},
+        "tenantId": {{ '\"' + azure_tenantId + '\"' }}
+      }
+- apiVersion: hive.openshift.io/v1
+  kind: ClusterImageSet
+  metadata:
+    name: {{ image_set|default("rhods-openshift") }}
+    namespace: {{ hive_claim_ns }}
+  spec:
+    releaseImage: {{ openshift_image|default("quay.io/openshift-release-dev/ocp-release:4.10.42-x86_64") }}
```

Comment:
I think it is time to update this..i know it doesn't matter but 4.10 is EOL

Reply:
done